src/aotcompile.cpp

```diff
@@ -309,7 +309,11 @@ void *jl_create_native(jl_array_t *methods, const jl_cgparams_t cgparams)
     JL_GC_POP();
 
     // process the globals array, before jl_merge_module destroys them
+#if JL_LLVM_VERSION >= 110000
+    std::vector<StringRef> gvars;
```

Comment:
This StringRefs get freed below (it's why we were using the implicit copy-constructor before—now you need to write the cast explicitly)